src/main/java/core/basesyntax/ColorsEnum.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum ColorsEnum {
```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
public enum Color {
```